consensus/tests/state_live_sync.rs

```diff
@@ -939,6 +956,7 @@ async fn clears_buffer_after_macro_block() {
         ),
         "Should accept chunks"
     );
+    yield_now().await;
```

Comment:
I don't think it's a good idea to rely on this function for our tests: https://docs.rs/tokio/1.38.0/tokio/task/fn.yield_now.html says that it's not guaranteed that the runtime will schedule other tasks on a `yield_now`. Could we maybe instead move this test to a multithreaded runtime?